### PR TITLE
Magazines and magazine issues collections

### LINKED ISSUE #157: Magazines and magazine issues collections
Magazines are, for example, Alkorytmi and recruitment magazine. Magazines have issues (releases) multiple times per year. Magazines are mostly PDFs, but can be other formats too.

TLDR:
- Page that displays a collection of PDFs
- One for Alkorytmi and one for recruitment

### PR DESCRIPTION
**Pull request by anolpe**
Closes #157

### CODE REVIEW

**Review by MikaelSiidorow — APPROVED** [review]
Looks solid! :moyai:

File: apps/cms/src/payload.config.ts [hunk 1]
```diff
@@ -224,6 +230,16 @@ export default buildConfig({
             baseURL: AZURE_STORAGE_ACCOUNT_BASEURL ?? "",
           }),
         },
+        [Documents.slug]: {
+          disableLocalStorage: true,
+          adapter: azureBlobStorageAdapter({
+            connectionString: AZURE_STORAGE_CONNECTION_STRING ?? "",
+            containerName: AZURE_STORAGE_CONTAINER_NAME ?? "",
```
[the remark this answers is not in the document]
Comment (anolpe) [reply apps/cms/src/payload.config.ts:237]: Maybe, but I don't know how that works:D

[the remark this answers is not in the document]
Comment (kahlstrm) [reply apps/cms/src/payload.config.ts:237]: Perhaps could be ideal to have different storage containers for these, i.e. `AZURE_MEDIA_STORAGE_CONTAINER_NAME` and `AZURE_DOCUMENTS_STORAGE_CONTAINER_NAME`

[the remark this answers is not in the document]
Comment (kahlstrm) [reply apps/cms/src/payload.config.ts:237]: This is to avoid collision of file names etc. inside the storage container

**Pull request merged**